styles.css

```diff
@@ -76,6 +75,6 @@
     grid-area: footer;
 }
 
-#blanco{
+#blanco {
```

Comment:
Deze code is overbodig. Je hebt al een ". " gedefinieerd in je grid template for main-e.